src/components/Markdown/MarkdownPreview.tsx

```
@@ -0,0 +1,18 @@
+
+type MarkdownPreviewProps = Readonly<{
+  html: SafeHtml;
+  className?: string;
+}>;
+
+export default function MarkdownPreview({
+  html,
+  className,
+}: MarkdownPreviewProps) {
+  return (
+    <div className={className} dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)